www-client/falkon/falkon-9999.ebuild

```diff
@@ -63,7 +63,7 @@ src_prepare() {
 
 src_configure() {
 	local mycmakeargs=(
-		-DDISABLE_DBUS=$(usex dbus)
+		-DDISABLE_DBUS=$(usex dbus no yes)
```

Comment:
thanks, just use `usex !dbus` instead ;)

Reply:
OK, I didn't know that. Feel free to redo that bit and merge as you wish, I don't have any real interest in that package.